### ISSUE TITLE
[$80][MSFT-187] Sample challenge page: Keyboard focus does not move to the popup and is not trapped in the challenge prerequisites popup.

### ISSUE THREAD

**Issue by veshu** [topcoder-platform/community-app#2855]
**Steps to Reproduce**
1. Go to the sample challenge page: https://www.topcoder.com/challenges/30095347.
 2. Activate the 'register' button'.
 3. There is a poppup which appears with details about the challenge prerequisites.
4. Note that keyboard focus doesn't move to the poppup and is not trapped inside this poppup, instead we are able to access the elements in the main page.

**Expected Result**
Keyboard focus should move to the poppup and should not be able to access the elements in the main page till the poppup is dismissed.

**Actual Result**
Keyboard focus doesn't move to the poppup and is not trapped inside the poppup.

**Device**: Desktop/Labtop
**Operating System**: Windows
**Browser**: Chrome

**WCAG Category**: 2. Operable
**WCAG Level**: A
**WCAG Success Criterion**: 2.1.1 Keyboard
**Screen Reader Used**: JAWS
**Accessibility Audit Tool used**: N/A

**Attachment**
https://topcodermsft-my.sharepoint.com/personal/pd-topcoder_topcodermsft_onmicrosoft_com/Documents/Forms/All.aspx?cid=088f5f5d%2D9b1c%2D4b7e%2Dacbf%2D320c257c9a9a&FolderCTID=0x0120005C598A51049FC14CBA882E1AEE168F51&id=%2Fpersonal%2Fpd%2Dtopcoder%5Ftopcodermsft%5Fonmicrosoft%5Fcom%2FDocuments%2FMSFT%2DTeams%2DQA%2FJune%202019%2F29%20Jun%2FTest%20Results%2FBug%20Videos%20%26%20Log%20Files%2FKameshwarid%2Fbug%2033

**Comment by ThomasKranitsas** [comment]
Contest https://www.topcoder.com/challenges/30095988 has been created for this ticket.<br/><br/>```This is an automated message for thomaskranitsas via Topcoder X```

**Comment by ThomasKranitsas** [comment]
Contest https://www.topcoder.com/challenges/30095988 has been updated - the new changes has been updated for this ticket.<br/><br/>```This is an automated message for thomaskranitsas via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30095988 has been updated - the new changes has been updated for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30095988 has been updated - it has been assigned to sadathanwar17.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by asadath1395** [comment]
@veshu Can you open this test challenge https://www.topcoder.com/challenges/30095347 or provide me other challenge id where i can reproduce this?
Thanks

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096891 has been created for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096891 has been updated - it has been assigned to sadathanwar17.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096891 has been updated - it has been assigned to Timor.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by romitgit** [comment]
How to activate the register button ?

**Comment by crazyk07** [comment]
We will make a new test challenge for this and I will post the link shortly.  @romitgit  Do you still want to try to fix this?  You will have to create a fake test account in order to replicate the same conditions.

@veshu  Please follow up on this later.  If no response, let's open this back for pickup.

**Comment by veshu** [comment]
@topcoder-platform/topcodercompetitors this is open for pickup.

**Comment by crazyk07** [comment]
Contest https://www.topcoder.com/challenges/30096891 has been updated - it has been assigned to rohitgupta_.<br/><br/>```This is an automated message for crazyk via Topcoder X```

**Comment by r0hit-gupta** [comment]
@veshu can you please provide with a new test challenge for testing? Thanks!

**Comment by crazyk07** [comment]
https://www.topcoder.com/challenges/30098777

**Comment by veshu** [comment]
@r0hit-gupta https://take.ms/MRQaP When tab is opening I can still navigate in header section of main page with tab, see above attached video.

**Comment by r0hit-gupta** [comment]
@veshu I have fixed the issue. Please review.

**Comment by drasticdpk** [comment]
@veshu @r0hit-gupta  I am unable to test it. Please provide test url with credential .

**Comment by r0hit-gupta** [comment]
@drasticdpk to test this, First I created a new test account on accounts.topcoder.com and then ran the app in production mode. After that, I tried registering for a challenge and saw the terms popup.

**Comment by drasticdpk** [comment]
Thank you @r0hit-gupta . @veshu  @sunildahiya does code has moved in prod env? Since we are unable to test this issue on dev env.

**Comment by veshu** [comment]
@sushilshinde ☝️

**Comment by tosha5252** [comment]
@veshu is there usually a pop-up after registering?  I can't reproduce this in prod.

**Comment by r0hit-gupta** [comment]
@tosha5252 check this once https://github.com/topcoder-platform/community-app/blob/develop/docs/mocking-terms.md
You should be able to mock the popup using this.

**Comment by drasticdpk** [comment]
@tosha5252 @r0hit-gupta , it was verified and working as expected.

**Comment by crazyk07** [comment]
Payment task has been updated: https://software.topcoder.com/review/actions/ViewProjectDetails?pid=30096891<br/><br/>```This is an automated message for crazyk via Topcoder X```
